src/everart/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@modelcontextprotocol/server-everart",
+  "version": "0.1.0",
```

Comment:
These version numbers must be in line with the other version numbers, since all release them in parallel